root/static/scripts/edit/URLCleanup.js

```diff
@@ -2642,6 +2644,36 @@ const CLEANUPS = {
       return url.replace(/^(?:https?:\/\/)?(?:[^.]+\.)?recochoku\.jp\/(album|artist|song)\/([a-zA-Z0-9]+)(\/)?.*$/, 'http://recochoku.jp/$1/$2/');
     },
   },
+  'residentadvisor': {
+    match: [
+      new RegExp('^(https?://)?(www\\.)?ra\\.co/', 'i'),
+    ],
```

Comment:
It would be nice to still match old-style `residentadvisor.net` URLs and do something helpful of it: Ideally convert them to the current style `ra.co` if there is a clear mapping between old style and current style.

Reply:
I thought about it, but given it seems they all redirect, it doesn't seem worth the effort. At most, maybe we could detect *any* residentadvisor.net URL and ask the user to follow the redirect?

Reply:
I did that in a separate commit.